RecoTracker/TrackProducer/test/ProbQXYAna.cc

```diff
@@ -0,0 +1,61 @@
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/PatCandidates/interface/IsolatedTrack.h"
+
+#include "DataFormats/TrackReco/interface/SiPixelTrackProbQXY.h"
+
+class ProbQXYAna : public edm::one::EDAnalyzer<edm::one::SharedResources> {
```

Comment:
this can be `global` too, as you don't have internal states to update. In any case what's the `SharedResources` that you are using here? I don't see any.